WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeSearchFragment.java

```diff
@@ -27,8 +27,8 @@
     public static ThemeSearchFragment newInstance() {
         return new ThemeSearchFragment();
     }
+    private static String mLastSearch = "";
```

Comment:
Earlier I thought of using a static variable instead of using savedInstanceState. Changing that